notebooks/5-stellar-flares/interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>seems like the <code>conda run </code>worked ok?</p><p> </p><p>if so, then we can add a link to <a href="https://outerspace.stsci.edu/display/MASTDOCS/Managing+Software" target="_blank">Managing Software</a> up above under "installation instructions"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>@ttdu yes it did! I'll be sure to add that link.</p>